tomviz/SaveLoadStateReaction.cxx

```diff
@@ -110,11 +114,34 @@ bool SaveLoadStateReaction::loadState(const QString& filename)
     }
   }
 
+  QDialog dialog(tomviz::mainWidget(), Qt::WindowStaysOnTopHint);
+  dialog.setModal(true);
+  dialog.setWindowTitle("Please wait...");
+
+  auto layout = new QHBoxLayout;
+  auto label =
+    new QLabel("Please wait... loading state file in progress", &dialog);
+  layout->addWidget(label);
+  dialog.setLayout(layout);
+  dialog.show();
+  // At least locally I need 2 processEvents calls for the dialog to show
+  // and appear non-blank.
+  QApplication::processEvents();
+  QApplication::processEvents();
```

Comment:
Even with these it doesn't seem to show up for me.

Reply:
This is unlikely to work as expected on all OSes, and we need to structure this so that is uses signals and slots. It might be better to split off the modal dialog from this PR, but I don't think this is the right way of adding a modal dialog that loops on processEvents until it thinks everything is complete.